app/db.py

```diff
@@ -40,8 +61,20 @@ def save_stock_data(data, symbol):
             )
         conn.commit()
 
+
 def drop_ohlcv_table():
     """ drop ohlcv data but only when using the test database """
     assert os.environ.get('ENV_MODE') == 'test', 'Not using test database, but trying to drop the ohlvc table'
     with conn.cursor() as cursor:
         cursor.execute("DELETE FROM ohlcv")
+
+
+def need_recent_data(symbol):
+    """ if we have data from today in the db we don't need more recent data"""
+    today = datetime.today()
+    day = date(today.year, today.month, today.day)
+    delta_days =  day - get_start_date(symbol)
+    if delta_days != 0: 
+        return True 
+    else:
+        return False
```

Comment:
omg I am so annoyed with my brain, I am always doing this with the True/ False when the statement already evaluates to that.